back-end/build.gradle

```diff
@@ -4,7 +4,7 @@ plugins {
 	id 'io.spring.dependency-management' version '1.1.0'
 }
 
```

Comment:
group name was corrected to 'com.ac3' could possibly clean up a couple empty lines other than that looks good